docs/source/contributing/documentation.rst

```diff
@@ -91,7 +91,7 @@ Example:
 
 See an example on list item 4.
 
-.. NOTE::
+.. note::
```

Comment:
What's the reason this changed? I'm not familiar with .rst type documentation.

Reply:
Apparently that is how we should write the note block in .rst files
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
Yep, `.. note::` is right, see also  here: https://manimce--1389.org.readthedocs.build/en/1389/contributing/documentation.html#note

Reply:
Great, thanks for cleaning up this document.